File: customer-address-jpa/src/main/java/com/izertis/example/repository/jpa/CustomerRepository.java
package com.izertis.example.repository.jpa;

import com.izertis.example.domain.Customer;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

/** Spring Data JPA repository for the Customer entity. */
@SuppressWarnings("unused")
@Repository
public interface CustomerRepository extends JpaRepository<Customer, Long> {

}

File: kustomer-address-jpa/src/main/kotlin/com/izertis/example/repository/jpa/CustomerRepository.kt
package com.izertis.example.repository.jpa

import com.izertis.example.domain.Customer
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/** Spring Data JPA repository for the Customer entity. */
@Repository interface CustomerRepository : JpaRepository<Customer, Long> {}
